src/layers/microscopy-viewer-layer/data-utils/tiff-utils.js

```diff
@@ -1,50 +1,64 @@
+/* eslint-disable no-await-in-loop */
```

Comment:
Is this necessary? "each successive operation will not start until the previous one has completed"

Reply:
No not necessary. Clumsy on my part. sorry for leaving this in (and the one below). Good catch.